api/estados/saopaulo.js

```diff
@@ -1,652 +1,654 @@
+    'Americana',
+    'Américo Brasiliense',
+    'Américo de Campos',
+    'Amparo',
+    'Analândia',
+    'Andradina',
+    'Angatuba',
+    'Anhembi',
+    'Anhumas',
+    'Aparecida',
+    "Aparecida d'Oeste",
+    'Apiaí',
+    'Araçariguama',
+    'Araçatuba',
+    'Araçoiaba da Serra',
+    'Aramina',
+    'Arandu',
+    'Arapeí',
+    'Araraquara',
+    'Araras',
+    'Arco-Íris',
+    'Arealva',
+    'Areias',
+    'Areiópolis',
+    'Ariranha',
+    'Artur Nogueira',
+    'Arujá',
+    'Aspásia',
+    'Assis',
+    'Atibaia',
+    'Auriflama',
+    'Avaí',
+    'Avanhandava',
+    'Aconsté',
+    'Bady Bassitt',
+    'Balbinos',
+    'Bálsamo',
+    'Bananal',
+    'Barão de Antonina',
+    'Barbosa',
+    'Bariri',
+    'Barra Bonita',
+    'Barra do Chapéu',
+    'Barra do Turvo',
+    'Barretos',
+    'Barrinha',
+    'Barueri',
+    'Bastos',
+    'Batatais',
+    'Bauru',
+    'Bebedouro',
+    'Bento de Abreu',
+    'Bernardino de Campos',
+    'Bertioga',
+    'Bilac',
+    'Birigui',
+    'Biritiba-Mirim',
+    'Boa Esperança do Sul',
+    'Bocaina',
+    'Bofete',
+    'Boituva',
+    'Bom Jesus dos Perdões',
+    'Bom Sucesso de Itararé',
+    'Borá',
+    'Boracéia',
+    'Borborema',
+    'Borebi',
+    'Botucatu',
+    'Bragança Paulista',
+    'Braúna',
+    'Brejo Alegre',
+    'Brodowski',
+    'Brotas',
+    'Buri',
+    'Buritama',
+    'Buritizal',
+    'Cabrália Paulista',
+    'Cabreúva',
+    'Caçapava',
+    'Cachoeira Paulista',
+    'Caconde',
+    'Cafelândia',
+    'Caiabu',
+    'Caieiras',
+    'Caiuá',
+    'Cajamar',
+    'Cajati',
+    'Cajobi',
+    'Cajuru',
+    'Campina do Monte Alegre',
+    'Campinas',
+    'Campo Limpo Paulista',
+    'Campos do Jordão',
+    'Campos Novos Paulista',
+    'Cananéia',
+    'Canas',
+    'Cândido Mota',
+    'Cândido Rodrigues',
+    'Canitar',
+    'Capão Bonito',
+    'Capela do Alto',
+    'Capiconsti',
+    'Caraguatatuba',
+    'Carapicuíba',
+    'Cardoso',
+    'Casa Branca',
+    'Cássia dos Coqueiros',
+    'Castilho',
+    'Catanduva',
+    'Catiguá',
+    'Cedral',
+    'Cerqueira César',
+    'Cerquilho',
+    'Cesário Lange',
+    'Charqueada',
+    'Chavantes',
+    'Clementina',
+    'Colina',
+    'Colômbia',
+    'Conchal',
+    'Conchas',
+    'Cordeirópolis',
+    'Coroados',
+    'Coronel Macedo',
+    'Corumbataí',
+    'Cosmópolis',
+    'Cosmorama',
+    'Cotia',
+    'Cravinhos',
+    'Cristais Paulista',
+    'Cruzália',
+    'Cruzeiro',
+    'Cubatão',
+    'Cunha',
+    'Descalvado',
+    'Diadema',
+    'Dirce Reis',
+    'Divinolândia',
+    'Dobrada',
+    'Dois Córregos',
+    'Dolcinópolis',
+    'Dourado',
+    'Dracena',
+    'Duartina',
+    'Dumont',
+    'Echaporã',
+    'Eldorado',
+    'Elias Fausto',
+    'Elisiário',
+    'Embaúba',
+    'Embu',
+    'Embu-Guaçu',
+    'Emilianópolis',
+    'Engenheiro Coelho',
+    'Espírito Santo do Pinhal',
+    'Espírito Santo do Turvo',
+    'Estiva Gerbi',
+    "Estrela d'Oeste",
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)